lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -11,6 +11,7 @@ import { Account } from "./Account";
 import { SSOTypes, Constants, PromptState, BlacklistedEQParams, libraryVersion } from "./utils/Constants";
 import { ClientConfigurationError } from "./error/ClientConfigurationError";
 import { StringUtils } from "./utils/StringUtils";
+import { UserSetRequestParams } from "./UserAgentApplication";
```

Comment:
If we stick with this type, I think it should be defined in this file, not UAA (as we want fewer things in UAA).

Reply:
Since we are passing it in `getCachedTokenInternal`. If you are going to fix it in angular 6/7/8 I can move the whole thing to `ServerRequestParameters`. There is no `request` parameter in   `getCachedTokenInternal` and it gets complicated. Only other option is to keep sending the old params as is and I find that not ideal either.